packages/amesos2/example/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 
-# TRIBITS_ADD_EXECUTABLE_AND_TEST(
-#   SimpleSolveExample
-#   SOURCES SimpleSolve
-#   ARGS
-#   COMM serial mpi
-# )
+TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  SimpleSolveExampleNonContigMap
+  SOURCES SimpleSolveNonContigMap
+  ARGS
+  COMM serial mpi
+)
```

Comment:
Isn't there a NUMPROCS variable where you specify the procs ? Otherwise CTest will fail with this 2 proc tests.